### ISSUE TITLE
Add comments in a pdf

### ISSUE THREAD

**Issue by ladyrick** [sumatrapdfreader/sumatrapdf#453]
I need a feature which allow me to add comment in a pdf.
It's very important.
Thank you

**Comment by rikvdh** [comment]
:+1: for this feature. It is for me the only reason I need to keep Adobe Reader side-by-side with Sumatra.

**Comment by odahan** [comment]
:+1:  One more for this _essential _feature. It is a bit a shame to keep adobe reader just for this feature. 
I know that SummatraPDF first advantage is to load fast and be **simple and efficient**, the goal is not the make it as fat as AR. But I know there are highlights in latest builds. This is not so important. Allowing to add notes will be really the only thing that is missing. Promise, I'll not ask for more ! :-)

**Comment by adamralph** [comment]
The lack of commenting is the only reason I don't use Sumatra.

**Comment by vyoussofzadeh** [comment]
I use Mendeley for commenting on a PDF. 
ps. I am a fan of SummatraPDF; it is a fantastic tool for integration with Latex.

**Comment by GitHubRulesOK** [comment]
@kjk duplicate of Comment feature #528

**Comment by SumatraPeter** [comment]
@GitHubRulesOK: As per your suggestion #591 was closed as a dupe of this one. So now instead of closing this one in turn as a dupe of #528 and creating an unnecessary chain, we can instead close #528 as a dupe and keep this one as the master. What do you think?

While we're at it might as well close #772 also as a dupe of this one.

**Comment by GitHubRulesOK** [comment]
@kjk 
whilst annotations are still a work of art in progress, this adding of text comments and free text appears to be currently "functional" (even if it could do with a tweak or two)
![image](https://user-images.githubusercontent.com/12754913/86522849-def32700-be5b-11ea-8a5a-0008918b8c82.png)

**Comment by patchbank** [comment]
What is the current state of this feature?

**Comment by GitHubRulesOK** [comment]
Current status is possibly classed as just above basic e.g.. better than most browser implimentations.
Highlight of text and areas in PDF is better than in previous versions and includes progressive saving / removal rather than the ability to also store in a seperate file (a feature I liked as it could be used with text files ebooks etc)
Upgraded to allow reviewers to  underline (2 types) / strike out text (previously required modifying the smx file)
Upgraded so reviewers comments and text notes can also be added / removed (but free text is currently a fixed size)
Posibility to move and delete some other annotations especially those produced by MuPDF

In summary passably good enough for me to use in my workplace productions instead of the slower company Acrobat

**Comment by adamralph** [comment]
How do I enable annotations? All I see in my context menu is this:

![image](https://user-images.githubusercontent.com/677704/92328078-aab80480-f05e-11ea-8375-1727aeff7b8d.png)

**Comment by GitHubRulesOK** [comment]
Annotation as "Work in Progress" has always only been available in the "Pre-release" versions thus features may come and go Highlight was available many years ago and worked well enough but did not include comments, Comments are working to what I consider as 95% requested and by including "notes" it is 100% fit for initial use (even if there may be other wishes/requirements re: usability (see seperate issue about color default etc.) For "notes" you can clearly use multiple free text boxes, but I feel the ability to resize (which again is a seperate issue for multiple annotations) is the last shortcoming for this request.

**Comment by jarvis-owl** [comment]
@adamralph 
Seems not to be included in the release version yet.

feel same as @rikvhd - really need this feature in a pdf viewer

**Comment by benyaminl** [comment]
Hello, I want to ask, there's AnnotationDefaults settings in pre-release/daily build, but seems since it's not released yet, seems there're no docs about it, the question that I want to ask, does it has any ability to change the default window size, position, of the anotation window? Is is there anyway in the future to support comment click to edit instead of right click edit anotation? Thanks

**Comment by GitHubRulesOK** [comment]
@kjk 
Since 3.3 is imminent and OP request is in effect addressed.
Any secondary requests above e.g. defaults etc. are covered by or needs be raised as separate issues against 3.3.
Thus this old one can now be closed?

**Comment by kjk** [comment]
agreed
